recipe/meta.yaml

```diff
@@ -25,23 +25,55 @@ requirements:
   host:
     - ncurses                # [not win]
     - readline               # [not win]
-  run:
-    - ncurses                # [not win]
-    - readline               # [not win]
 
-test:
-  commands:
-    - sqlite3 --version
-    - if not exist %PREFIX%\\Library\bin\sqlite3.dll exit 1    # [win]
-    - if not exist %PREFIX%\\Library\lib\sqlite3.lib exit 1    # [win]
-    - test -f $PREFIX/lib/libsqlite3.dylib                     # [osx]
-    - test -f $PREFIX/lib/libsqlite3.so                        # [linux]
-    - if not exist %PREFIX%\\Library\include\sqlite3.h exit 1  # [win]
-    - test -f $PREFIX/include/sqlite3.h                        # [not win]
+
+# By compiling with readline, the /bin/sqlite3 executable becomes
+# subject to the GPL
+# Therefore, we decide to split it to make it clear
+# That the other software is not.
+outputs:
+  - name: libsqlite
+    files:
+      - lib                      # [not win]
+      - share                    # [not win]
+      - include                  # [not win]
+      - Library/lib/             # [win]
+      - Library/include/         # [win]
+      - Library/bin/sqlite3.dll  # [win]
+    requirements:
+      - ncurses                   # [notwin]
+    test:
+      commands:
+        - if not exist %PREFIX%\\Library\bin\sqlite3.dll exit 1    # [win]
+        - if not exist %PREFIX%\\Library\lib\sqlite3.lib exit 1    # [win]
+        - test -f $PREFIX/lib/libsqlite3.dylib                     # [osx]
+        - test -f $PREFIX/lib/libsqlite3.so                        # [linux]
+        - if not exist %PREFIX%\\Library\include\sqlite3.h exit 1  # [win]
+        - test -f $PREFIX/include/sqlite3.h                        # [not win]
+    about:
+      home: http://www.sqlite.org/
+      license: Public-Domain (http://www.sqlite.org/copyright.html)
+      summary: 'Library for implementing a self-contained, zero-configuration, SQL database engine.'
+  - name: sqlite
+    files:
+      - bin/sqlite3               # [not win]
+      - Library/bin/sqlite3.exe   # [win]
+    requirements:
+      run:
+        - {{ pin_subpackage("libsqlite", exact=true) }}
+        - readline   # [notwin]
+    test:
+      commands:
+        - sqlite3 --version
+    about:
+      home: http://www.sqlite.org/
+      license: GPL-3.0  # [not win]
```

Comment:
I don't think that it is OK to change the license here.

Reply:
You mean the win/not win flag?

Reply:
I mean that I don't think it is OK to advertise that a license covers a package that the original authors did not intend

Reply:
That is, readline is clearly GPL'd and so SQLite should retain its license